src/components/Board.js

```diff
@@ -13,6 +12,20 @@ class Board extends Component {
         '','','','','','','','',''
       ],
       winner: false,
+      shouldRefresh: false
+    }
+  }
+
+  componentWillReceiveProps() {
+    if (this.props.shouldRefresh) {
```

Comment:
If I get a prop `shouldRefresh` that is `true`, I run this code. I set the state to the initial Board state.